marimo/_server/sessions.py

```diff
@@ -1065,6 +1064,42 @@ def get_active_connection_count(self) -> int:
             ]
         )
 
+    # Check every 1 minute.
+    # We check the TTL later, so the 1 minute check is just to avoid
+    # unnecessary work.
+    @debounce(60)
+    def cleanup_orphaned_sessions(self) -> None:
```

Comment:
Perplexing bug; I would have thought that

https://github.com/marimo-team/marimo/blob/0d719dd64756f3eb10c73a62193fb404166fde7a/marimo/_server/api/endpoints/ws.py#L468-L488

handled this case. Unless somehow the websocket's on_disconnect is not called?